### PR TITLE
Fix player screen throw an error when any click is made

### PR DESCRIPTION
**Pull request by Sceat**
✅ Closes: #365

### CODE REVIEW

**Comment by Sceat** [comment]
@DeltaEvo i can't understand the pipeline 🤯

File: src/context.js [hunk 1]
```diff
@@ -245,7 +245,7 @@ export async function observe_client(context) {
   player_health.observe(context)
   player_attributes.observe(context)
   player_chat.observe(context)
-  player_screen.observe(context)
+  // player_screen.observe(context)
```
Comment (DeltaEvo) [inline src/context.js:248]: You don't need to remove that

File: src/context.js [hunk 2]
```diff
@@ -81,10 +81,10 @@ const world_reducers = [
...
   register_player_traders,
   register_player_deal_damage,
-  register_screen({
-    id: 'player_screen',
-    size: { width: 8, height: 4 },
-  }),
+  // register_screen({
+  //   id: 'player_screen',
+  //   size: { width: 8, height: 4 },
+  // }),
```
Comment (DeltaEvo) [inline src/context.js:87]: Remove it instead of comment it

**Review by DeltaEvo — CHANGES_REQUESTED** [review]

**Comment by Sceat** [comment]
resolved but what the heck is wrong with the pipeline 🤯

**Comment by DeltaEvo** [comment]
What do you mean by pipeline ?

**Comment by Sceat** [comment]
the CI

**Review by DeltaEvo — APPROVED** [review]

**Pull request merged**